stacks/LinkdexStack.ts

```diff
@@ -10,23 +10,25 @@ export function LinkdexStack({ app, stack }: StackContext) {
     : new Bucket(stack, 'cars')
 
   const customDomain = getCustomDomain(app.stage, process.env.HOSTED_ZONE)
+
+  const lindexKeyFn = new Function(stack, 'LindexKeyFn', {
+    handler: 'functions/linkdex.handler',
+    permissions: [bucket],
+    environment: { BUCKET_NAME: bucket.bucketName },
+    architecture: 'arm_64', // cheaper, so why not?
+    timeout: '15 minutes',  // default is 10s. Api gateway has 30s limit. Lambda max 15 mins.
+    url: true
```

Comment:
This is the magic config option that gets aws to assign a direct lambda url for the function.

Reply:
but you can't set it via api gateway config, so we create our own function construct here and pass it to the api.

Reply:
who wow, interesting that using the "correct way of doing aws" with gateway in front leads to such a smaller timeout